skaha/client.py

```diff
@@ -30,43 +31,49 @@ class MyClient(SkahaClient):
 
     """
 
-    server = attrib(default="https://ws-uv.canfar.net/skaha")
-    certificate = attrib(
-        default="{HOME}/.ssl/cadcproxy.pem".format(HOME=environ["HOME"]), type=str
+    server: str = Field(default="https://ws-uv.canfar.net/skaha", title="Server URL")
+    certificate: str = Field(
+        default="{HOME}/.ssl/cadcproxy.pem".format(HOME=environ["HOME"]), type=str,
```

Comment:
You should perform a validation check to see if the certificate exists and is readable.

Reply:
You can use the `FilePath` pydantic type here.